### PR TITLE
Add xeus-octave

### PR DESCRIPTION
**Pull request by marimeireles**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by marimeireles** [comment]
@conda-forge-admin, please rerender

**Comment by SylvainCorlay** [comment]
Let's chat about this on Monday!

**Comment by marimeireles** [comment]
:D

**Comment by marimeireles** [comment]
:kissing_closed_eyes:

File: recipes/xeus-octave/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#!/bin/bash
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX
```
Comment (xhochy) [inline recipes/xeus-octave/build.sh:4]: We supply a set of sane defaults here that you can use instead.
```suggestion
cmake ${CMAKE_ARGS}
```

Comment (marimeireles) [reply recipes/xeus-octave/build.sh:4]: Hum! Didn't know about that, thank you.

**Comment by marimeireles** [comment]
hey @xhochy since you already started have a look here, could this get merged? :)

File: recipes/xeus-octave/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    - glfw
+    - octave
+
+test:
+  requires:
+    - jupyter_kernel_test
+  commands:
+    - echo ${PREFIX}
+    - ls ${PREFIX}
+    - ls ${PREFIX}/bin/
```
Comment (xhochy) [inline recipes/xeus-octave/meta.yaml:51]:
```suggestion
```

File: recipes/xeus-octave/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    - xorg-libxfixes  # [linux]
+    - glfw
+    - libpng
+  run:
+    - glfw
+    - octave
+
+test:
+  requires:
+    - jupyter_kernel_test
```
Comment (xhochy) [inline recipes/xeus-octave/meta.yaml:47]: This doesn't seem to be used here.

File: recipes/xeus-octave/build.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#!/bin/bash
+
+cmake ${CMAKE_ARGS}
+
+make install -j8
```
Comment (xhochy) [inline recipes/xeus-octave/build.sh:6]:
```suggestion
ninja install
```

File: recipes/xeus-octave/build.sh [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+#!/bin/bash
+
+cmake ${CMAKE_ARGS}
```
Comment (xhochy) [inline recipes/xeus-octave/build.sh:4]: Use ninja to make use of all CPUs (better)

```suggestion
cmake -GNinja ${CMAKE_ARGS}
```

File: recipes/xeus-octave/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pkg-config  # [osx]
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
```
Comment (xhochy) [inline recipes/xeus-octave/meta.yaml:26]:
```suggestion
    - ninja
```

File: recipes/xeus-octave/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    - jupyter_kernel_test
+  commands:
+    - echo ${PREFIX}
+    - ls ${PREFIX}
+    - ls ${PREFIX}/bin/
+    - test -f ${PREFIX}/bin/xeus-octave  # [unix]
+
+about:
+  home: https://github.com/rapgenic/xeus-octave
+  license: GPL
```
Comment (xhochy) [inline recipes/xeus-octave/meta.yaml:56]: The linter isn't happy with this, please find out the correct one according to SPDX. You can remove `license_familiy` afterwards, that will be autofilled if this one is correct.

**Review by xhochy — COMMENTED** [review]
Left you a bunch of comments to address :)

**Comment by marimeireles** [comment]
maybe now? :)
@xhochy

**Comment by marimeireles** [comment]
Thank you for the help @xhochy! :)
Lovely evening :cherry_blossom:

**Pull request merged**